docs/core-components/OpenAPI-Specs.yaml

```
@@ -0,0 +1,533 @@
+    description: Development server
+  - url: https://auth.example.com/api
+    description: Production server
+
+components:
+  securitySchemes:
+    sessionAuth:
+      type: apiKey
+      in: cookie
+      name: connect.sid
+      description: Session-based authentication via HTTP-only cookies
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)